__tests__/fixtures/tailwind-input-with-explicit-screen-utilities.css

```diff
@@ -0,0 +1,11 @@
+@responsive {
+    .example {
+        color: red;
+    }
+}
+
+@tailwind screen-utilities;
+
+.john {
+    content: "wick";
+}
```

Comment:
This test case is exactly what I was going to suggest! 😄 Nice work!

Reply:
:-D Testcase should be good now.
I'll make it green now ;-)

Reply:
@adamwathan Yeah, test is green now. Could you review again and please do not hesitate to correct me again or give mit hints where to improve.